components/keyboard_shortcuts/keyboard_shortcuts_sequence/keyboard_shortcuts_sequence.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {FormatXMLElementFn, PrimitiveType} from 'intl-messageformat';
+import React, {memo} from 'react';
+import {useIntl} from 'react-intl';
+
+import {isMessageDescriptor, KeyboardShortcutsSeq} from '../keyboard_shortcuts';
+
+import {isMac} from '../../../utils/utils';
+import './keyboard_shortcuts_sequence.scss';
+import {ShortcutKetVariant, ShortcutKey} from '../../shortcut_key';
+
+type Props = {
+    shortcut: KeyboardShortcutsSeq;
+    values?: Record<string, PrimitiveType | FormatXMLElementFn<string, string>>;
+    hideDescription?: boolean;
+    hoistDescription?: boolean;
+    isInsideTooltip?: boolean;
+};
+
+function normalize(shortcut: KeyboardShortcutsSeq) {
+    if (isMessageDescriptor(shortcut)) {
+        return shortcut;
+    }
+
+    const {default: standard, mac} = shortcut;
+    return isMac() && mac ? mac : standard;
+}
+
+const KEY_SEPARATOR = '|';
+
+function KeyboardShortcutSequence({shortcut, values, hideDescription, hoistDescription, isInsideTooltip}: Props) {
+    const {formatMessage} = useIntl();
+    const shortcutText = formatMessage(normalize(shortcut), values);
+    const splitShortcut = shortcutText.split('\t');
+
+    let description: string | undefined;
+    let keys: string | undefined;
```

Comment:
there is no need to define types for those variables

Reply:
typescript will start complaining about type being implicitly ``any``, So i guess it make sense to keep